spec/unit/hanami/cli/commands/gem/new_spec.rb

```diff
@@ -1,15 +1,22 @@
 # frozen_string_literal: true
 
 RSpec.describe Hanami::CLI::Commands::Gem::New do
-  subject {
+  subject do
     described_class.new(bundler: bundler, out: out, fs: fs, inflector: inflector)
-  }
+  end
+
+  # TODO: remove this block
```

Comment:
Just checking that you intend for this TODO to be kept here?

Reply:
@timriley Good catch: removed. https://github.com/hanami/cli/pull/99/commits/c89e6f84263ba6c076af9cf198e0113752d2acae